### PR TITLE
[Remove Stream/Optional usage] LPS-170733 Remove usages of Optional on Method Return Value for module segments-asah-connector | part 1

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:tango

**Comment by W2jx** [comment]
ci:test:tango

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java [hunk 1]
```diff
@@ -99,20 +96,21 @@ public void checkIndividualSegments(long companyId, String individualPK)
...
+		for (String individualSegmentId : individualSegmentIds) {
+			SegmentsEntry segmentsEntry =
+				_segmentsEntryLocalService.fetchSegmentsEntry(
+					serviceContext.getScopeGroupId(), individualSegmentId,
+					true);
+
+			if (segmentsEntry != null) {
+				segmentsEntryIds.add(segmentsEntry.getSegmentsEntryId());
+			}
+		}
```
Comment (yuhai) [inline modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java:111]: @W2jx,  please replace the for loop with TransformUtil.transform.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:tango

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/128207

**Pull request closed without merging**